test_deepcopy_objective fails

https://github.com/ICB-DCM/pyPESTO/actions/runs/12026858059/job/33526668472:

```
___________________________ test_deepcopy_objective ____________________________

    def test_deepcopy_objective():
        """Test copying objectives (needed for MultiProcessEngine)."""
        petab_importer = pypesto.petab.PetabImporter.from_yaml(
            os.path.join(
                models.MODELS_DIR,
                "Boehm_JProteomeRes2014",
                "Boehm_JProteomeRes2014.yaml",
            )
        )
        factory = petab_importer.create_objective_creator()
        objective = factory.create_objective()
    
        objective.amici_solver.setSensitivityMethod(
            amici.SensitivityMethod_adjoint
        )
    
        objective2 = copy.deepcopy(objective)
    
        # test some properties
        assert (
            objective.amici_model.getParameterIds()
            == objective2.amici_model.getParameterIds()
        )
        assert (
            objective.amici_solver.getSensitivityOrder()
            == objective2.amici_solver.getSensitivityOrder()
        )
        assert (
            objective.amici_solver.getSensitivityMethod()
            == objective2.amici_solver.getSensitivityMethod()
        )
        assert len(objective.edatas) == len(objective2.edatas)
    
        assert objective.amici_model is not objective2.amici_model
        assert objective.amici_solver is not objective2.amici_solver
>       assert objective.steadystate_guesses is not objective2.steadystate_guesses
E       AttributeError: 'AmiciObjective' object has no attribute 'steadystate_guesses'. Did you mean: 'reset_steadystate_guesses'?

test/base/test_engine.py:114: AttributeError
```

--- a/test/base/test_engine.py
+++ b/test/base/test_engine.py
@@ -86,7 +86,14 @@ def test_deepcopy_objective():
         )
     )
     factory = petab_importer.create_objective_creator()
-    objective = factory.create_objective()
+    amici_model = factory.create_model()
+    amici_model.setSteadyStateSensitivityMode(
+        amici.SteadyStateSensitivityMode.integrateIfNewtonFails
+    )
+    amici_model.setSteadyStateComputationMode(
+        amici.SteadyStateComputationMode.integrateIfNewtonFails
+    )
+    objective = factory.create_objective(model=amici_model)
 
     objective.amici_solver.setSensitivityMethod(
         amici.SensitivityMethod_adjoint

--- a/test/petab/test_amici_objective.py
+++ b/test/petab/test_amici_objective.py
@@ -86,12 +86,21 @@ def test_preeq_guesses():
     importer = pypesto.petab.PetabImporter.from_yaml(
         os.path.join(models.MODELS_DIR, model_name, model_name + ".yaml")
     )
-    problem = importer.create_problem()
-    obj = problem.objective
-    obj.amici_solver.setNewtonMaxSteps(0)
+    obj_creator = importer.create_objective_creator()
+    amici_model = obj_creator.create_model()
+    amici_model.setSteadyStateComputationMode(
+        amici.SteadyStateComputationMode.integrateIfNewtonFails
+    )
+    amici_model.setSteadyStateSensitivityMode(
+        amici.SteadyStateSensitivityMode.integrateIfNewtonFails
+    )
+    obj = obj_creator.create_objective(model=amici_model)
+    problem = importer.create_problem(objective=obj)
     obj.amici_model.setSteadyStateSensitivityMode(
         amici.SteadyStateSensitivityMode.integrationOnly
     )
+    obj = problem.objective
+    obj.amici_solver.setNewtonMaxSteps(0)
     obj.amici_solver.setAbsoluteTolerance(1e-12)
     obj.amici_solver.setRelativeTolerance(1e-12)
 

